crates/matrix-sdk-crypto/src/utilities.rs

```diff
@@ -113,6 +113,11 @@ where
         Duration::from_secs(delay)
     }
 
+    /// Add a single item to the cache.
+    pub fn insert(&self, item: T) {
+        self.extend([item].into_iter());
```

Comment:
`extend` usually takes an `IntoIterator`. I first thought that this was from the `Extend` trait which enforces this, but looks like we have our own inherent method. Want to change that function so we don't need `.into_iter()` in this place?

Reply:
Sure: https://github.com/matrix-org/matrix-rust-sdk/pull/1970/commits/37361359313556259a4e7531966e1cc50e660fd2.